.travis.yml

```diff
@@ -5,12 +5,13 @@ php:
  - 7.0
  - 7.1
  - 7.2
+ - 7.3
 # Commands to be run before your environment runs.
 before_script:
  - composer self-update
- - composer install --prefer-source --no-interaction --dev
```

Comment:
- Remove `--prefer-source` option because we should install the stable dependency versions.
- Remove `--dev` option because it's deprecated.